src/redis_ipc.c

```diff
@@ -168,21 +166,21 @@ int redis_ipc_init(const char *this_component, const char *this_thread)
     // each thread gets its own results queue in redis
     memset(result_queue_path, 0, sizeof(result_queue_path));
     ret = ipc_path(result_queue_path, sizeof(result_queue_path),
-                  RPC_TYPE_RESULT, this_component, this_thread);
+                   RPC_TYPE_RESULT, this_component, this_thread);
     if (ret != RIPC_OK)
         goto redis_ipc_init_failed;
     new_info->result_queue_path = strdup(result_queue_path);
     if (new_info->result_queue_path == NULL)
         goto redis_ipc_init_failed;
 
-    redis_ipc_info = new_info; 
+    redis_ipc_info = new_info;
 
     return RIPC_OK;
 
 redis_ipc_init_failed:
     if (stderr_debug_is_enabled())
     {
-            fprintf(stderr, "[ERROR] redis_ipc_init failed for thread %s\n", this_thread);
+        fprintf(stderr, "[ERROR] redis_ipc_init failed for thread %s\n", this_thread);
```

Comment:
I'm not sure about what "Added line #L183" means; is the syntax picky enough to care about a trailing space?  is it still valid with the changes?

Reply:
Yes, codecov seems to consider this a whole new line because it was "touched" by the whitespace patch.